contents/blog/posthog-vs-pendo.md

```diff
@@ -0,0 +1,651 @@
+---
+date: 2023-04-25
+title: "In-depth: PostHog vs Pendo"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/posthog-vs-pendo/posthog-vs-pendo.jpg
+featuredImageType: full
+author:
+    - joe-martin
+category: General
+tags:
+    - Comparisons
+    - Guides
+---
+
+Want to understand the difference between Pendo and PostHog? Here's the short answer:
+
+1. Pendo enables users to add in tool-tips and in-app messages. It includes some product analytics tools.
+
+2. PostHog is an all-in-one product improvement platform that includes the ability to add in-app messages, as well as analytics, feature flags, session replays and more.
+
+Now it's time for the long answer...
+
+In this article we’ll explore the crucial differences and similarities between Pendo (specifically, Pendo Engage) and PostHog. We’ll cover topics such as:
+
+- Pendo and PostHog’s [core features](#core-features)
+- [Product analytics](#product-analytics) and [in-app messaging](#in-app-messages-and-prompts) features in detail
+- [Integrations](#apps-and-integrations) and [data pipelines](#data-pipelines)
+- [Libraries](#library-support), [tracking differences and SDKs](#sdks-and-tracking)
+- [Privacy, security and regulatory compliance](#compliance-and-security)
+- [Pricing details](#how-much-does-pendo-cost), [free trials](#do-pendo-and-posthog-offer-free-trials) and other [frequently asked questions](#frequently-asked-questions)
+
+## How is PostHog different?
+
+### 1. PostHog is an all-in one platform
+PostHog brings all the tools engineers need to measure success, run experiments, and more, into one platform. It’s a complete, all-in-one product OS, with robust analytics, feature flagging, A/B testing, and session capturing features. Pendo, on the other hand, is a more limited, and requires you to adopt other platforms, such as HotJar or LaunchDarkly, to get comparable functionality to PostHog.
+
+### 2. PostHog is built for engineers
+We built PostHog to support technically-savvy product managers and engineers — especially [engineers with a product focus in their role](/blog/what-is-a-product-engineer). As such, PostHog includes many powerful features that aren’t commonly available in tools like Pendo, which are built for more general audiences.
+
+### 3. PostHog is open source
+Because we build for engineers first, we're also open source. You can check out [PostHog’s source code](https://github.com/PostHog), [build integrations](/docs/apps/build) or [other services](/blog/how-we-built-an-app-server) on top of the product, and even [give feedback or interact with the team via GitHub](https://github.com/PostHog).Transparency, including transparent pricing, is one of our core values.
```

Comment:
```suggestion
Because we build for engineers first, we're also open source. You can check out [PostHog’s source code](https://github.com/PostHog), [build integrations](/docs/apps/build) or [other services](/blog/how-we-built-an-app-server) on top of the product, and even [give feedback or interact with the team via GitHub](https://github.com/PostHog). Transparency, including transparent pricing, is one of our core values.
```